### ISSUE TITLE
Move to synchronous `db.load`

### ISSUE THREAD

**Issue by twolfson** [dresende/node-orm2#598]
Inside of `db.load`, there is no reason for it to be an asynchronous function. We should make it synchronous and deprecate the `cb` functionality.

```js
db.load(__dirname + '/path/to/models.js');
```

**Comment by dxg** [comment]
I agree. I don't use `.load` personally, and load models my own way:
```js
files = "model1 model2 model3".split(' ')
for (var i = 0; i < files.length; i++) {
  require("./" + files[i])(orm, db);
}
```

**Comment by rainabba** [comment]
If the models only load once and can't be used until they've loaded, synchronous (blocking) actually makes sense here. If the app requires the DB, then instead of having to check if it's initialized in every call, I'd rather block during app load. I'm facing a similar issue with another ORM.
